social_recovery/contracts/SocialRecoveryModule.sol

```diff
@@ -0,0 +1,148 @@
+pragma solidity >=0.5.0 <0.7.0;
+import "@gnosis.pm/safe-contracts/contracts/base/Module.sol";
+import "@gnosis.pm/safe-contracts/contracts/base/ModuleManager.sol";
+import "@gnosis.pm/safe-contracts/contracts/base/OwnerManager.sol";
+import "@gnosis.pm/safe-contracts/contracts/common/Enum.sol";
+
+
+/// @title Social Recovery Module - Allows to replace an owner without Safe confirmations if friends approve the replacement.
+/// @author Stefan George - <stefan@gnosis.pm>
```

Comment:
I would adjust this to include yourself and me (as the original version of stefan looked quite different ;) )